backend/src/schema/resolvers/follow.spec.js

```diff
@@ -131,6 +131,21 @@ describe('follow', () => {
       })
     })
 
+    test('adds `createdAt` to `FOLLOW` relationship', async () => {
+      await mutate({
+        mutation: mutationFollowUser,
+        variables,
+      })
+      const relation = await neode.cypher(
+        'MATCH (user:User {id: {id}})-[relationship:FOLLOWS]->(followed:User) WHERE relationship.createdAt IS NOT NULL RETURN relationship',
+        { id: 'u1' },
+      )
+      const relationshipProperties = relation.records.map(
+        record => record.get('relationship').properties.createdAt,
+      )
+      expect(relationshipProperties[0]).toEqual(expect.any(String))
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-10-18T13:51:22Z" title="Friday, October 18th 2019, 3:51:22 pm +02:00">Oct 18, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -129,6 +129,16 @@ describe('follow', () => {
         data: { followUser: expectedUser },
         errors: undefined,
       })
+
+      // Test to make sure FOLLOWS relationship has "createdAt" date.
+      const relation = await neode.cypher(
+        'MATCH (user:User {id: {id}})-[relationship:FOLLOWS]->(followed:User) WHERE relationship.createdAt IS NOT NULL RETURN relationship',
+        { id: 'u1' },
+      )
+      const relationshipProperties = relation.records.map(
+        record => record.get('relationship').properties.createdAt,
+      )
+      expect(relationshipProperties[0]).toEqual(expect.any(String))
```


can you please place this into it's own `it` block? 
we should avoid multiple `expect` in the same it block

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-18T13:52:19Z" title="Friday, October 18th 2019, 3:52:19 pm +02:00">Oct 18, 2019</time>_

---


Usually you would try to have one assertion per spec. You can easily create a new spec by saying:
```js
it('adds `createdAt` to `FOLLOW` relationship', async () => { ... })
```

Redundancy is not a problem with software tests.

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-10-18T13:54:13Z" title="Friday, October 18th 2019, 3:54:13 pm +02:00">Oct 18, 2019</time>_

---


you can run `await mutate({ mutation: mutationFollowUser, variables })` then this code

Reply:
> [<img alt="KapilJ22" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/KapilJ22) **Authored by [KapilJ22](https://github.com/KapilJ22)**
_<time datetime="2019-10-20T19:22:00Z" title="Sunday, October 20th 2019, 9:22:00 pm +02:00">Oct 20, 2019</time>_

---


@mattwr18 -  Much appreciated your review. Implemented changes as you requested.

Reply:
> [<img alt="KapilJ22" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/KapilJ22) **Authored by [KapilJ22](https://github.com/KapilJ22)**
_<time datetime="2019-10-20T19:23:35Z" title="Sunday, October 20th 2019, 9:23:35 pm +02:00">Oct 20, 2019</time>_

---


@roschaefer I like your suggestion - "Redundancy is not a problem with software tests." I was thinking in terms of code.
Much appreciated your review. Implemented changes as you requested.